geyser-plugin-manager/src/geyser_plugin_manager.rs

```diff
@@ -221,12 +225,27 @@ impl GeyserPluginManager {
         let mut current_plugin = self.plugins.remove(idx);
         let name = current_plugin.name().to_string();
         current_plugin.on_unload();
+        // The plugin must be first dropped before dropping the library to avoid crash.
```

Comment:
Please just PR this separately, as it's not related to this changeset.

Reply:
okay will move to another PR